packages/calcite-components/src/components/flow/flow.e2e.ts

```diff
@@ -55,6 +56,24 @@ describe("calcite-flow", () => {
       expect(flowItem).toBeNull();
     });
 
+    it("should call setFocus() on back button click", async () => {
+      const page = await newE2EPage();
+
+      await page.setContent(
+        html`<calcite-flow
+          ><calcite-flow-item id="one"></calcite-flow-item><calcite-flow-item id="two"></calcite-flow-item
+        ></calcite-flow>`
+      );
+
+      await page.$eval("#two", (elm: HTMLCalciteFlowItemElement) =>
+        elm.dispatchEvent(new CustomEvent("calciteFlowItemBack"))
+      );
+      await page.waitForTimeout(0);
```

Comment:
Curious, do we still need the timeout delay after the `setFocus` changes? If so, maybe we could add a new util (separately) to ensure we wait consistently for `setFocus`-related tests.

Reply:
Seems to not be necessary here. Ill remove.